Kinvey/Kinvey/ReadOperation.swift

```diff
@@ -63,6 +63,18 @@ extension ReadOperationType {
                 request.addRequest(self.executeNetwork(completionHandler))
             }
             return AnyRequest(request)
+        case .networkOtherwiseLocal:
+            let requests = MultiRequest<Result<SuccessType, FailureType>>()
+            let networkRequest = executeNetwork() {
+                switch $0 {
+                case .success:
+                    completionHandler?($0)
+                case .failure:
+                    requests.addRequest(self.executeLocal(completionHandler))
```

Comment:
Need to check why we failed here because we only want to fetch from the local datastore if a network connectivity issue occurred. If a 4xx/5xx response comes back, we still need to correctly handle this exception by alerting the user.